info/templates/info/dashboard_event.html

```diff
@@ -14,14 +14,14 @@
 {% endblock %}
 
 {% block content %}
-    <div class="row">
+    <div class="row px-5">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
```

Comment:
This isn't necessary, it's already included in the base.html file: https://github.com/nmunro/resourcedb/blob/7d92f6edb269c40a93d92cf356a34e3636e7f6f7/info/templates/base.html#L9, please remove this line :)